src/main.py

```diff
@@ -35,15 +35,20 @@ class SongList:
     songs = []
     current_song_number : int
     def __init__(self, 
+            searchstr = None,
             song = None, 
             playlist = None, 
             shuffle = False, 
             repeatqueue = False,
-            repeatone = False):
+            repeatone = False,
```

Comment:
Is Running now ? you added a extra comma at the end of last argument of function